libkmod/libkmod-builtin.c

```diff
@@ -69,6 +70,11 @@ static struct kmod_builtin_iter *kmod_builtin_iter_new(struct kmod_ctx *ctx)
 		goto fail;
 	}
 
+	if (sb.st_size > INTPTR_MAX) {
```

Comment:
There is an implicit cast in line 86, because `iter.size` is a `ssize_t` variable.
https://github.com/kmod-project/kmod/pull/96/files#diff-0ff4aa938ae2e1d5acdda6004a5fb28dc6a21c820fc70aa2dea1787a63c60961R86`

Reply:
As stated, `iter.size` is `ssize_t`, i.e. signed. There is no `SSIZE_MAX`, so `INTPR_MAX` gets closest to it.